src/js/dataMapping/shared/recipientIdentifiers.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import GlossaryLink from "components/sharedComponents/GlossaryLink";
+import { DUNS_LABEL } from "GlobalConstants";
+
+/**
+ * Create array of entity IDs (UEI and/or DUNS) formatted for Additional Information sections
+ * @param {string} duns DUNS number (if any)
+ * @param {string} uei UEI (if any)
+ * @returns array of string(s) with value, label and glossary link
+ */
+export const idList = (duns, uei) => {
+    const ids = [];
+    if (uei) {
+        ids.push(
+            <>
+                {uei} (UEI <GlossaryLink term="Unique Entity Identifier (UEI)" />)
```

Comment:
The glossary opens but  the term UEI isn't selected.  The GlossaryLink term prop should be "unique-entity-identifier-uei"